src/main/java/se/sundsvall/seabloader/api/InvoicesResource.java

```diff
@@ -43,4 +42,14 @@ public ResponseEntity<Void> createInvoice(@NotNull @RequestBody final byte[] bod
 		invoiceService.create(body);
 		return noContent().build();
 	}
+
+	@PostMapping(path = "/exports", produces = APPLICATION_PROBLEM_JSON_VALUE)
```

Comment:
Move this to own resource class (/schedulers or likewise) to separate (and possibly also hide) resources that maybe should not be available to all clients?